pkgs/overlay.nix

```diff
@@ -79,12 +79,34 @@ in {
 
   mailx = super.callPackage ./mailx.nix { };
   mc = super.callPackage ./mc.nix { };
-  mongodb_3_2 = super.callPackage ./mongodb/3.2.nix {
+
+  mongodb-3_2 = super.callPackage ./mongodb/3.2.nix {
     sasl = super.cyrus_sasl;
     boost = super.boost160;
     # 3.2 is too old for the current libpcap version 1.9, use 1.8.1
     libpcap = self.libpcap_1_8;
   };
+  mongodb_3_2 = self.mongodb-3_2;
+  mongodb-3_4 = super.mongodb;
```

Comment:
There is a chance that this breaks during future updates of our nixpkgs fork and mongodb-3_4 will contain something different than 3.4.x. Better guard this assumption in the test.